pkgs/development/python-modules/python-vagrant/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, vagrant, fetchPypi, lib }:
+buildPythonPackage rec {
+  version = "0.5.15";
+  pname = "python-vagrant";
+
+  src = fetchPypi {
+      inherit pname version;
+      sha256 = "1ikrh6canhcxg5y7pzmkcnnydikppv7s6sm9prfx90nk0ac8m6mg";
+  };
+
+  checkInputs = [ vagrant ];
```

Comment:
```suggestion
```
You should instead patch the executable path here: https://github.com/todddeluca/python-vagrant/blob/master/vagrant/__init__.py#L128-L129